docs/www/configuration.md

```diff
@@ -625,6 +625,20 @@ rpk:
   # Default: ''
   coredump_dir: "/var/lib/redpanda/coredump"
 
+  # Creates a "ballast" file that can be removed in the odd case that a redpanda node
+  # runs out of space, so that the node can continue to accept commands while a user
+  # deletes a topic or deletes records.
```

Comment:
```suggestion
  # delete a topic or records to reduce the memory used by Redpanda.
```

Reply:
I changed "memory" for "space", since this is about storage. Thanks for the suggestions!